.changeset/itchy-ghosts-begin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Added warning usecase for all ProgressIndicators
```

Comment:
Also, since `ProgressIndicator` is not a component, you can either say "progress indicators" or `ProgressRadial`/`ProgressLinear`.

Reply:
should this be edited with a new changeset or by editing the existing manually?

Reply:
You can manually make this change in the same changeset file.

Also, in case writing changeset messages in the terminal is not preferrable, you can also add a dummy placeholder message when running `yarn changeset` in the terminal and then edit the file manually after the CLI creates it with the dummy message.

Reply:
alright yeah. updated it!